build/common.sh

```diff
@@ -737,27 +739,36 @@ function kube::release::sha1() {
 function kube::release::create_docker_images_for_server() {
   # Create a sub-shell so that we don't pollute the outer environment
   (
+    local binary_dir="$1"
     local binary_name
-    for binary_name in "${KUBE_DOCKER_WRAPPED_BINARIES[@]}"; do
+    for wrappable in "${KUBE_DOCKER_WRAPPED_BINARIES[@]}"; do
+
+      local oldifs=$IFS
+      IFS=","
+      set $wrappable
+      local binary_name="$1"
+      local base_image="$2"
+      IFS=$oldifs
```

Comment:
I'm googling a solution, and I can't find a cleaner one then good old save and restore. I could use a regex or expand the entry into an array but the options I'm coming up with are generally grosser. If you have suggestions, I will gladly hear them. Bash is gross :)